internal/service/cloudbackupschedule/resource_cloud_backup_schedule_test.go

```diff
@@ -507,7 +524,25 @@ func configDefault(info *acc.ClusterInfo, p *admin.DiskBackupSnapshotSchedule) s
 	`, info.ClusterNameStr, info.ProjectIDStr, p.GetReferenceHourOfDay(), p.GetReferenceMinuteOfHour(), p.GetRestoreWindowDays())
 }
 
-func configCopySettings(projectID, clusterName string, p *admin.DiskBackupSnapshotSchedule) string {
+func configCopySettings(projectID, clusterName string, emptyCopySettings bool, p *admin.DiskBackupSnapshotSchedule) string {
+	var copySettings string
+	if emptyCopySettings {
+		copySettings = ""
```

Comment:
default value for copySettings is "", you can have just the else as the if condition:

if !emptyCopySettings { ...

Reply:
Thanks updated in f9d2e6964d64617b60dc4145853fe9c329533b68